src/secure_random.cr

```diff
@@ -1,19 +1,60 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom library is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, and uuid, etc.
```

Comment:
I will suggest replace *and uuid, etc.* with *uuid and others* to avoid confusions around [Serial comma](https://en.wikipedia.org/wiki/Serial_comma) for non-native english speakers (is an hexadecimal string and uuid, or is hexadecimal string *and* also uuid).